src/router.js

```diff
@@ -0,0 +1,30 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+Vue.use(Router);
+
+export default new Router({
+  mode: 'history',
+  routes: [
+    {
+      path: '/',
+      name: 'MainMenu',
+      component: () => import('./views/MainMenu.vue')
+    },
+    {
+      path: '/alphabet',
+      name: 'Alphabet',
+      component: () => import('./views/Alphabet.vue')
```

Comment:
Maybe put files from url `/alphabet`
in `/views/alphabet/` directory?
When there will be more components, main: /views  directory will be more organized